components/toast/toast.css

```diff
@@ -0,0 +1,15 @@
+.toast {
+  align-items: center;
+  display: flex;
+  height: auto;
+  justify-content: space-between;
+  margin: auto;
+  max-width: 400px;
```

Comment:
not sure if you are sticking to rem unit, but you can use rem here to maintain responsiveness.

Reply:
Ya, changed that . Thanks for pointing out that.